lib/compat/wordpress-6.0/class-wp-theme-json-gutenberg.php

```diff
@@ -176,6 +176,7 @@ class WP_Theme_JSON_Gutenberg extends WP_Theme_JSON_5_9 {
 		'layout'          => array(
 			'contentSize' => null,
 			'wideSize'    => null,
+			'padding'     => null,
```

Comment:
Just want to make sure I understand the intention with this: I wasn't sure why we need to add an additional `padding` setting under the `layout` object. What's the difference between this, and the root level `spacing.padding` style value? From looking at the code, my guess is that this value is in settings, so it means it's a fallback value that can appear anywhere in the hierarchy rather than specifically meant to be applied to the root (or other block)?

If it causes confusion, is it worth contextualising this particular padding value with a prefix? (In much the same way as `contentSize` is an abstraction for `max-width` rather than calling it `max-width` directly)

Reply:
> What's the difference between this, and the root level spacing.padding style value?

The root level value only applies to the root container. Currently it gets added to `body` (#39926 tries to change that logic to allow direct children of root to be full width even when there's padding set).

This value applies to any block that has layout set to inherit default, no matter whereabouts in the nesting structure it may be.

Reply:
> This value applies to any block that has layout set to inherit default, no matter whereabouts in the nesting structure it may be.

Gotcha, thanks for clarifying!